scripts/load-env.js

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable @typescript-eslint/no-var-requires*/
```

Comment:
I added this so that the crowdin.yml app can look for the env variables from this location when running the scripts from package.json.